deploy.sh

```diff
@@ -1,31 +1,4 @@
 #!/bin/bash
```

Comment:
i also wasn't sure if we want to move completely to make, also for the deployment